### PR TITLE
feat: add jwtSecret to app.conf & edit application page

### PR DESCRIPTION
**Pull request by 1340908470**
- Display **jwtSecret** in "Applications - Edit Application page", to make sure developer could use jwtSecret just as clientSecret when init config before using sdk.
- Move jwtSecret string to app.conf to make sure developer could easily find it
- Add jwtSecret's length to fit to [JJWT](https://github.com/jwtk/jjwt)

### CODE REVIEW

**Comment by hsluoyz** [comment]
@sh1luo @Kininaru plz review

File: conf/app.conf [hunk 1]
```diff
@@ -2,12 +2,12 @@ appname = casdoor
 httpport = 8000
 runmode = dev
 SessionOn = true
+jwtSecret = "asdhasoidasdiojasioduwadoinsadiauwdhalwduhnodhcnaw"
 copyrequestbody = true
 driverName = mysql
-dataSourceName = root:123@tcp(localhost:3306)/
+dataSourceName = cwh:Cwh2001128@tcp(rm-2ze1ie6967guwf9z1io.mysql.rds.aliyuncs.com:3306)/
 dbName = casdoor
```
Comment (sh1luo) [inline conf/app.conf:9]: why use your own db info?

File: conf/app.conf [hunk 2]
```diff
@@ -2,12 +2,12 @@ appname = casdoor
...
-dataSourceName = root:123@tcp(localhost:3306)/
+dataSourceName = cwh:Cwh2001128@tcp(rm-2ze1ie6967guwf9z1io.mysql.rds.aliyuncs.com:3306)/
 dbName = casdoor
 redisEndpoint =
 authState = "casdoor"
 httpProxy = "127.0.0.1:10808"
-verificationCodeTimeout = 10
-initScore = 2000
\ No newline at end of file
+verificationCodeTimeout = 10
```
Comment (sh1luo) [inline conf/app.conf]: why remove this field?

File: web/src/ApplicationEditPage.js [hunk 3]
```diff
@@ -218,6 +218,14 @@ class ApplicationEditPage extends React.Component {
...
         </Row>
+        <Row style={{marginTop: '20px'}} >
+          <Col style={{marginTop: '5px'}} span={(Setting.isMobile()) ? 22 : 2}>
+            {Setting.getLabel(i18next.t("provider:Jwt secret"), i18next.t("provider:Jwt secret - Tooltip"))} :
+          </Col>
+          <Col span={22} >
+            <Input value={this.state.application.jwtSecret} disabled />
+          </Col>
+        </Row>
         <Row style={{marginTop: '20px'}} >
```
Comment (sh1luo) [inline web/src/ApplicationEditPage.js:229]: Either through editing `conf/app.conf` , or through the UI(such as xxxEdit Page), both appear is a bit redundant

**Pull request closed without merging**